frontend/src/notApp.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import Header from './Components/Header/Header';
+import './App.css';
+import CommunityProjects from './Components/CommunityProjects/CommunityProjects';
+import CommunityPosts from './Components/CommunityPosts/CommunityPosts';
+import { projectList } from './projectList';
+import { posts } from './posts';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      route: 'home',
+      user: '',
+      projectList: projectList,
+      posts: posts,
+    }
+  }
+
+  onRouteChange = (route, index) => {
```

Comment:
react-router-dom (React Router) allows for routes to be created in a cleaner fashion. It eliminates having to pass a route change function to each component.

https://reacttraining.com/react-router/core/api/Route

Reply:
This file has been deleted